CHANGELOG.md

```diff
@@ -3,11 +3,12 @@ All notable changes to this project will be documented in this file.
 This project adheres to [Semantic Versioning](http://semver.org/).
 Use the following format for additions: ` - VERSION: [feature/patch (if applicable)] Short description of change. Links to relevant issues/PRs.`
 
-- 1.1.16 (not release yet):
```

Comment:
Hm I see the intention here, but we shouldn't do this. Every push to master should have a new version number and a corresponding line in the changelog, no matter how small.